### DISCUSSION TITLE
Antenna Selection

### DISCUSSION THREAD

**Discussion by conner-replogle** [Comet-Robotics/URC#6] [Networking]
We need to select an antenna to send video feeds and other large amounts of data over/under 1km of range without LOS

**Comment by conner-replogle** [comment]
"Taking a look at the Toronto robot, for their on rover comms they look to be using an ubiquity bullet titanium with an omni vertical antenna
(titanium doesn't look to be sold anymore, but other bullet models are https://dl.ubnt.com/datasheets/bulletm/BulletM_Ti_DS.pdf)
It doesn't look like ubiquity themselves sell a compatible antenna, all of theirs have 2 SMA ports instead of one n-connector port, but its a simple enough design with a standard enough connector" -Alice

![image2](https://github.com/user-attachments/assets/c8332ee4-1901-4b2e-bfb7-140d367049ce)
![image](https://github.com/user-attachments/assets/2c263244-45dd-42d5-a191-e5045c053158)


"We tested our 2.4GHz dish (nanobridge m2) with a cell phone, got degraded but good enough signal from ecsw to sciences 
With a better radio on the client side and a location with a lower noise floor range shouldn't be an issue 
But, the signal fully dropped at any hint of a loss of line of sight"- Alice

A client antenna better than an cell phone will hopefully perform better. Need to find one to test with sometime in the following weeks.

**Comment by CrispyAlice2** [comment]
On the rover side, ideally an omnidirectional antenna with a radiation pattern focused on the horizon should be used.
For the radio, [Ubiquity Bullet M2 ](https://store.ui.com/us/en/products/bulletm2-hp) is a good option, it's essentially the same as the one Toronto Met used.
For the antenna, it will need to have an n type connector. Looking on Amazon, there seems to be a huge range in prices but I can't figure out exactly why.
Example antenna: [link](https://www.amazon.com/Network-AOA-2458-59-TM-Dual-Band-Omni-Directional-Connector/dp/B01N4GLQYW)

**Comment by CrispyAlice2** [comment]
I was also gathering information on things we could stick on lower bands to get around small to medium line of sight blockers. My friend has used [adafruit Lora boards](https://www.adafruit.com/product/3073) to get 50kbps on the 70cm band (430MHz), which would be plenty for the essential command and control systems. 70cm band is also really easy to get amplifiers and other equipment for, I think K5UTD might already have some laying around in one of our cabinets. This avenue needs a slight bit more thought before buying anything, I'll keep y'all posted.

**Reply by conner-replogle** [reply]
We looked into LoRa Last semester would be interesting for a backup /fallback communication system

**Reply by CrispyAlice2** [reply]
47 CFR 97.313(j) means we are limited to 10W on lora since its a spread spectrum modulation, which is actually a lot for what we need to do. For context, the part 15 wifi stuff is less than half a watt. Being SS, it should be resistant to interference etc which is good.

**Comment by CrispyAlice2** [comment]
I did some more looking at some of the other robots, it looks like the ubiquity rocket line is also popular, but the 2.4 GHz version of this product is out of stock right now. Looks like the main advantage is that it can use "MIMO" to have a higher throughput, but it doesn't look like its any higher power or better sensitivity or anything. It would also mean rethinking the base station side in order to use mimo, probably not worth the mild upgrade.

**Comment by CrispyAlice2** [comment]
Here's the things that I think need to be bought.

Amazon:
1x [https://www.amazon.com/Ubiquiti-BULLET-M2-HP-Outdoor-802-11-M2HP/dp/B002SYS22E](https://www.amazon.com/Ubiquiti-BULLET-M2-HP-Outdoor-802-11-M2HP/dp/B002SYS22E)
1x [https://www.amazon.com/Network-AOA-2458-59-TM-Dual-Band-Omni-Directional-Connector/dp/B01N4GLQYW](https://www.amazon.com/Network-AOA-2458-59-TM-Dual-Band-Omni-Directional-Connector/dp/B01N4GLQYW)

Digikey:
2x [https://www.digikey.com/en/products/detail/adafruit-industries-llc/3073/6005358](https://www.digikey.com/en/products/detail/adafruit-industries-llc/3073/6005358)
2x [https://www.digikey.com/en/products/detail/cinch-connectivity-solutions-johnson/142-0711-841/390187](https://www.digikey.com/en/products/detail/cinch-connectivity-solutions-johnson/142-0711-841/390187)
